.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - TRAVIS_NODE_VERSION="10.10.0"
     - secure: "JPdkdywgQbUUixuDuATMXZHWpWImziRoGKd1tCjYd3+0lamLBlldmjhsZ+Bp7ZabKg2ExQZareGDp34cJdOwWBGb2gg0/emjy4UkWkbDH28zyoWNK1SFp6OTY8AeY/icPXKw3MBgKoXG+hemfcxhYc8rUNgUt7V2fA0JSAkCi4w="
     - secure: "BrGlbfGLxZYCynh7LeIFyQeiTx4YX6fdMYK2UceVLoAFjOml9mvtJY5i4Oafc2iBDPQ0wAP/eIzZOTcddwc+12o1S0EAq3zZb5z+MOwRKjsiC+BGUXNzAGLaeJqFTC5Sn+vvqsaDGaR3jrdQ8APUL7XapuLUAkof7vqc1SPsn3I="
+    - secure: "YH4s6huZYW34XFlKDAg7yC8VGUjIiird3gDKOmHyhlPVoQa75tll3iqVBBxNFcj+5CfzlCU+ZZK/JTvRoLJ7+QBPrRm7eZ6+s44JaJK+eAz12AJKdEfZyp+blMQ26Y8uOz5AZQhnRsoYfbqDK3p/T8FW1MAKdvE+8nHDCdAh26k="
```

Comment:
Does this travis key meed to be here given we are going with github actions?

Reply:
Nope. In fact that whole file can go away. Should I just do that now? I was going to do that in a separate PR, but if it's appropriate here, I'll nuke it.

Reply:
I am fine putting it in a separate ticket or in this PR. I think we have DATAUP-234 to remove old code, but its up to you.

Reply:
I'll just use that other ticket to remove the travis stuff and update that PR. Thanks for the feedback!